tests/Backend/Snowflake/DirectAccessTest.php

```diff
@@ -426,6 +437,62 @@ public function testDirectAccessAndRestrictions()
             . ' AS SELECT * FROM "in.c-API-DA_TEST"."other_table"',
             $views[0]['text']
         );
+
+        //Linked bucket
+        $this->_client->shareOrganizationProjectBucket($bucketId);
+
+        $client2DirectAccess = new DirectAccess($client2);
+        try {
+            $client2DirectAccess->deleteCredentials(self::BACKEND_SNOWFLAKE);
+        } catch (\Keboola\StorageApi\ClientException $e) {
+        }
+
+        $client2Credentials = $client2DirectAccess->createCredentials(self::BACKEND_SNOWFLAKE);
+
+        $response = $client2->listSharedBuckets();
+        $sharedBucket = reset($response);
+        $linkedBucketId = $client2->linkBucket(
+            $linkedBucketName,
+            $linkedBucketStage,
+            $sharedBucket['project']['id'],
+            $sharedBucket['id']
+        );
+
+        $client2DirectAccess->enableForBucket($linkedBucketId);
+
+        $client2Connection = new Connection([
+            'host' => $client2Credentials['host'],
+            'user' => $client2Credentials['username'],
+            'password' => $client2Credentials['password'],
+        ]);
+
+        $schemas = $client2Connection->fetchAll('SHOW SCHEMAS');
+
+        $this->assertCount(2, $schemas, 'There should be INFORMATION SCHEMA and one bucket');
+        $schemas = array_values(array_filter($schemas, function ($schema) {
+            return $schema['name'] === 'DA_IN_API-LINKED-TESTS';
+        }));
+        $this->assertSame('DA_IN_API-LINKED-TESTS', $schemas[0]['name']);
+
+        $client2Connection->query(sprintf(
+            'USE SCHEMA %s',
+            $client2Connection->quoteIdentifier($schemas[0]['name'])
+        ));
+        $viewsResult = $client2Connection->fetchAll('SHOW VIEWS');
+        $this->assertCount(1, $viewsResult);
```

Comment:
Tady nemá být 1, ale 2!

Reply:
Jo diky, mal som za to ze som si to pustal potom aj na komplet odkomentovanych testoch, chvalabohu by to spadlo na testoch, kazdopadne pustil som si testy a su tam dve tabulky 
![Screenshot 2020-06-04 at 21 35 28](https://user-images.githubusercontent.com/6448364/83802708-8fc6a480-a6ab-11ea-810b-1778baba7ac5.png)